test/e2e/common/setup-kubestellar.sh

```diff
@@ -86,27 +86,28 @@ echo "wds1 created."
 
 :
 : -------------------------------------------------------------------------
-: Run OCM transport controller executable
+: Run OCM transport controller in a pod
 :
 cd "${SRC_DIR}/../../.." ## go up to KubeStellar directory
 KUBESTELLAR_DIR="$(pwd)"
-## this is a temp solution - run as executble process. this should be replaced to run as pod, ideally using helm
-OCM_TRANSPORT_PLUGIN_RELEASE="0.1.0-rc2"
+OCM_TRANSPORT_PLUGIN_RELEASE="0.1.0-rc4"
 curl -sL https://github.com/kubestellar/ocm-transport-plugin/archive/refs/tags/v${OCM_TRANSPORT_PLUGIN_RELEASE}.tar.gz | tar xz
 cd ocm-transport-plugin-${OCM_TRANSPORT_PLUGIN_RELEASE}
 OCM_TRANSPORT_PLUGIN_DIR="$(pwd)"
 pwd
 echo "replace github.com/kubestellar/kubestellar => ${KUBESTELLAR_DIR}/" >> go.mod
-go mod tidy
-make build
-mv ./bin/ocm-transport-plugin ${KUBESTELLAR_DIR}/ocm-transport-plugin
+IMAGE_TAG=${OCM_TRANSPORT_PLUGIN_RELEASE} make ko-build-local
+kind load --name kubeflex docker-image ko.local/transport-controller:${OCM_TRANSPORT_PLUGIN_RELEASE} # load local image to kubeflex
 cd "${KUBESTELLAR_DIR}"
 pwd
 rm -rf ${OCM_TRANSPORT_PLUGIN_DIR}
```

Comment:
FYI: this is really painful when something goes wrong. How about moving this `rm` to the end of each test, where it is only done on success?